_pages/en/strategy-build-vs-buy.md

```diff
@@ -56,29 +31,30 @@ permalink: /strategy-build-vs-buy.html
 - [Coherent set of actions](#coherent-set-of-actions)
 - [Measuring the Strategy's success](#measuring-the-strategys-success)
 - [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
-- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix B - Traceability Matrix (Update coming soon)](#appendix-b---traceability-matrix-update-coming-soon)
 - [Appendix D - Definitions](#appendix-d---definitions)
-- [Appendix E - References](#appendix-e---references)
+- [Appendix E - Notes and Change Log](#appendix-e---notes-and-change-log)
+- [Appendix F - References](#appendix-f---references)
 
 ## Introduction
```

Comment:
Should there be a toggle near the top of this page to link to the 'Build vs. Buy - explained page'?

Reply:
Yes! Fixed that :grimacing: 

Moved it at the bottom by accident :facepalm: